viewing_party/party.py

```diff
@@ -1,23 +1,126 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dictionary = {}
+    if title and genre and rating:
+        movie_dictionary["title"] = title
+        movie_dictionary["genre"] = genre
+        movie_dictionary["rating"] = rating
+    else:
+        return None
+    
+    return movie_dictionary

```

Comment:
Great job with this method! Couple of small callouts here.

You could convey intent a little more explicitly by pulling "return None" out of the else condition and use a guard clause with:
```python 
if not title or not genre or not rating: 
    return None 

# then proceed to create your movie dict
```

I would also swap the order of your guard clause and where you initialize movie_dictionary since we won't even need movie_dictionary if the inputs are invalid and we need to return from the function quickly. 

This way, when another developer comes along, they can immediately recognize "Oh if these values aren't valid, then we'll return None. The way your code is currently structured, I can tell at some point this method will return None, but it's not as immediately clear why the method would return None. 

Also a note about guard clauses - The Guard Clause pattern helps in these cases by “inverting” that way of thinking and rapidly returning false conditions and keeping the important code blocks (usually the longest) in the lowest indentation level possible. Read more [here](https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3)

You can also create your dictionary with literal syntax and return it without first creating the variable movie_dictionary:

```python
def create_movie(title, genre, rating):
    if not title or not genre or not rating: 
        return None 

    return { "title": title, "genre": genre, "rating": rating }
```